stable/mysql/README.md

```diff
@@ -90,6 +90,7 @@ The following table lists the configurable parameters of the MySQL chart and the
 | `metrics.readinessProbe.timeoutSeconds`      | When the probe times out                                                                     | 1                                                    |
 | `resources`                                  | CPU/Memory resource requests/limits                                                          | Memory: `256Mi`, CPU: `100m`                         |
 | `configurationFiles`                         | List of mysql configuration files                                                            | `nil`                                                |
+| `service.annotations`                        | Kubernetes annotations                                                                       | {}                                                   |
```

Comment:
Could you specify that this is annotations for the service specifically?

Reply:
Thank you for all your feedback :)